### ISSUE TITLE
icon warning

### ISSUE THREAD

**Issue by jiwonman** [parsajiravand/vue-client-recaptcha#23]
I saw the simple mode with custom icon in the demo.

So I used <template #icon> to create my own custom image.
The image is displayed correctly, but I see the following warning in the F12 console:

If I don't customize the icon and use the default icon instead, the warning does not appear.
However, when I customize the icon, I get the following warning:

[Vue warn]: Non-function value encountered for default slot. Prefer function slots for better performance.

**Comment by jiwonman** [comment]
Additionally, I'm attaching my code.

```
<div class="sample-captcha">
  <section class="data-box">
    <input
      type="text"
      id="secure_text"
      v-model="secure_text"
      name="secure_text"
      class="w238"
    />
  </section>
  <section class="captcha-box">
    <VueClientRecaptcha
      :value="secure_text"
      @isValid="checkValidCaptcha"
      custom-text-color="#3c5684"
    >
      <template #icon>
        <span style="margin-left: 8%; cursor: pointer"
          ><img
            src="@/assets/images/btn/btn_refresh.gif"
            alt="refresh"
        /></span>
      </template>
    </VueClientRecaptcha>
  </section>
  &nbsp;&nbsp;
</div>
```

**Comment by parsajiravand** [comment]
Hello, Thanks a lot for your report,  I fix this warning in new version **"1.1.6"**.
Now you can update your package and use it without warning.
